### DISCUSSION TITLE
Dolphin not working properly

### DISCUSSION THREAD

**Discussion by Naimul007A** [prasanthrangan/hyprdots#2071] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/58e5f972-abe9-44b2-82ba-fad188a29afa)
my  Dolphin text show black suddenly .My theme aslo black type and i loved this.
How can i fixed this ?

**Comment by kRHYME7** [comment]
show as your .config/kdeglobals

**Reply by Naimul007A** [reply]
![image](https://github.com/user-attachments/assets/fc55e63d-3a4f-4b11-b0d8-16b89193a2e3)

**Reply by kRHYME7** [reply, marked as answer]
then check the kdeglobals of this repo. Add it to your configs.
